src/OutSystems/GridAPI/Auxiliary.ts

```diff
@@ -63,6 +63,32 @@ namespace OutSystems.GridAPI.Auxiliary {
     export function GetHashCode(str: string): number {
         return OSFramework.DataGrid.Helper.GenerateHashCode(str);
     }
+
+    /**
+     * Receives a function and its name. Executes it and measures it properly.
+     * @param functionName Name of the function that will be measured
+     * @param fn Function that will be measured and executed
+     * @returns Output of the received function
+     */
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    export function MeasurePerformance<T extends (...args: any[]) => any>(
```

Comment:
actually you could replace `any` with `unknown`, no @glundgren93?

Reply:
I tried that, but it raised another error, so I opted for this instead.